src/Backend/Modules/Groups/Entity/GroupActionRight.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace Backend\Modules\Groups\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * This is the GroupActionRight Entity
+ *
+ * @author Mathias Dewelde <mathias@dewelde.be>
+ *
+ * @ORM\Entity
+ * @ORM\Table(name="GroupActionRight")
```

Comment:
This is not entirely sure. If this table is only used in this module, it's completely safe (and advised) to use the new name)